common/yak/java/tests/member_test.go

```diff
@@ -28,4 +28,60 @@ class A {
 			ssaapi.WithLanguage(ssaapi.JAVA),
 		)
 	})
+
+	t.Run("set member use this", func(t *testing.T) {
+		code := `
+package foo.bar;
+class A {
+	public  int value;
+	public void set(int num) {
+		this.value = num;
+	}
+	public void get() {
+		return this.value;
+	}
+	
+	public static void main(){
+ 		A a = new A();
+		a.set(12);
+		println(a.get());
+	}
+}
+		`
+		ssatest.CheckSyntaxFlow(t, code,
+			`println(* as $result)`,
+			map[string][]string{
+				"result": {"Undefined-a.get(valid)(Undefined-A(Undefined-A)) member[side-effect(Parameter-num, this.value)]"},
+			},
+			ssaapi.WithLanguage(ssaapi.JAVA),
+		)
+	})
+
+	t.Run("set member does not use this", func(t *testing.T) {
+		code := `
+package foo.bar;
+class A {
+	public  int value;
+	public void set(int num) {
+		value = num;
+	}
+	public int get() {
+		return this.value;
```

Comment:
test `get() {return value;}`.
set is assign class member without `this`, get is read class member without `this`. 
test case should have:
* asign&read with this 
* only assign without this 
* only read without this 
* assign&read both without this 
* static method read variable that same name with class member (should not create undefine-this)